deploy/uninstall/uninstall.yaml

```diff
@@ -90,13 +90,13 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: longhorn-uninstall-service-account
-    namespace: default
+    namespace: longhorn-system
 ---
 apiVersion: batch/v1
 kind: Job
 metadata:
   name: longhorn-uninstall
-  namespace: default
+  namespace: longhorn-system
```

Comment:
I think the uninstall job was initially run outside of the `longhorn-system` namespace since it would also require cleaning up the namespace itself. And therefore there would be no way to see the status of the uninstall.

Is there any dependency on the HELM side that would require to know the status of the uninstall pod / job?

Reply:
> I think the uninstall job was initially run outside of the longhorn-system namespace since it would also require cleaning up the namespace itself. And therefore there would be no way to see the status of the uninstall.

The installation pod doesn't seem to make attempt to clean up the `longhorn-system` namespace from testing and code

> Is there any dependency on the HELM side that would require to know the status of the uninstall pod / job?

In Helm, the installation job has been running inside the same `longhorn-system` namespace so far. Foe Helm, the flow is currently (as will stay the same):
1. installation job runs as Helm `pre-delete` hook
2. Helm watches the job to be completed and start deleting manifests
3. The namespace still exist after Helm installation

Reply:
I think the uninstall job in the default namespace only exists in longhorn-manager repo, but if looking into the chart where the uninstall job there is actually just in the same namespace as the installed longhorn. 

So, it should be fine to use the same namespace to run the uninstall job. As I mentioned, the duplicated manifests will be removed from the longhorn-manager repo, also installing/uninstalling from longhorn-manager is not a formal way for users but rather for our internal development purpose only.